appcues/src/main/java/com/appcues/debugger/DebuggerStatusManager.kt

```diff
@@ -96,10 +96,11 @@ internal class DebuggerStatusManager(
         updateData()
     }
 
-    fun checkDeepLinkValidation(deepLinkPath: String) {
+    suspend fun checkDeepLinkValidation(deepLinkPath: String) {
         if (deepLinkPath == deepLinkValidationToken) {
             deepLinkConfigured = true
             deepLinkValidationToken = null
+            updateData()
```

Comment:
noticed that without this update _sometimes_ the state of the UI would not update to reflect the change appropriately.  I'm not sure how this was missed or working last time...

Reply:
its probably because one of the other items was triggering the update.